Python/BitCoin Mining/BitCoin_Mining.py

```diff
@@ -0,0 +1,74 @@
+# Import the sha256 function
+from hashlib import sha256
+
+# Nonce value 
+MAX_NONCE = 10000
+
+
+# Function for encoding text to a 64 bit hexadecimal value
+def SHA256(text):
+    """
+    >>> SHA256('Hello')
+    185f8db32271fe25f561a6fc938b2e264306ec304eda518007d1764826381969
+    >>> SHA256('Text')
+    71988c4d8e0803ba4519f0b2864c1331c14a1890bf8694e251379177bfedb5c3
+    """
+    return sha256(text.encode("ascii")).hexdigest()
+
+
+# function for guessing nonce value
+def mine(block_number, transactions, previous_hash, prefix_zeros,MAX_NONCE):

```

Comment:
Can you use proper docstrings here?

Reply:
I'm sorry I did not understand, can you suggest what could I Improve?
or where did I go wrong?
Thanks